### PR TITLE
Profile: Ripal Handoo

### LINKED ISSUE #5293: [Community] Member Profile: Ripul Handoo
**Current Behavior**

@RipulHandoo has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @RipulHandoo as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @RipulHandoo
- Twitter: 
- LinkedIn: [/in/ripul-handoo/](https://www.linkedin.com/in/ripul-handoo/)
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
**Description**

This PR fixes #5293 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by srujan0404** [comment]
@Vidip-Ghosh LGTM!!

**Comment by l5io** [comment]
🚀 Preview for commit 0abb49c6a8682cb9cab9260e6fb238d142f17d65 at: https://65a965c832768000a090a87c--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@Vidip-Ghosh have you confirmed the profile is visible because its not

File: src/collections/members/Ripal_Handoo/index.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Ripul Handoo
+position: Contributor and Community member
+image_path: ./Ripal_Handoo.webp
+github: RipulHandoo
+twitter: ripul_handoo
+linkedin: ripul-handoo
+layer5: a7b465ad-21eb-4f3f-9c84-56de76aebe62
+bio: Hello, I'm Ripul Handoo, currently pursuing my B.Tech in Computer Science and Engineering at VIT, Pune. I am passionate about learning new technologies and actively contribute to open-source projects.
+published: #true
```
Comment (sudhanshutech) [inline src/collections/members/Ripal_Handoo/index.mdx:10]: remove # from here, then it might
```suggestion
published: true
```

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by Vidip-Ghosh** [comment]
Its deployment is now failing @sudhanshutech

**Comment by sudhanshutech** [comment]
yes you can take care of it by making some small commit from your side, try it

**Comment by Vidip-Ghosh** [comment]
> yes you can take care of it by making some small commit from your side, try it

I did this but still preview failing

**Comment by iArchitSharma** [comment]
are you following all the steps - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#set-up-your-local-development-environment

**Comment by Vidip-Ghosh** [comment]
> are you following all the steps - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#set-up-your-local-development-environment

Yes I have followed

**Comment by l5io** [comment]
🚀 Preview for commit 21ea3efbf09bfda7ed4fb3ef2ec446d2bc507d11 at: https://65ba42342cc4cad426ac0030--layer5.netlify.app

**Comment by iArchitSharma** [comment]
@RipulHandoo all good?

**Comment by RipulHandoo** [comment]
Yes it looks good. Just a small change, my name is wrong in the image path.

**Comment by l5io** [comment]
🚀 Preview for commit a58122d21d0756f717af73d14c1579455e33c878 at: https://65ba9a5c34a6760dbce218fd--layer5.netlify.app

**Pull request merged**